README.md

```diff
@@ -106,6 +106,38 @@ JUDOSCALE = {
 
 Note the [official recommendations for configuring Flask](https://flask.palletsprojects.com/en/2.2.x/config/#configuration-best-practices).
 
+# Using Judoscale with Celery and Redis
+
+> **NOTE 1:** The Judoscale Celery integration currently only works with the [Redis broker](https://docs.celeryq.dev/en/stable/getting-started/backends-and-brokers/index.html#redis).
+
+> **NOTE 2:** Using [task priorities](https://docs.celeryq.dev/en/latest/userguide/calling.html#advanced-options) is currently not supported.
```

Comment:
I think we should clarify what "not supported" means here. My understanding is that users can still enqueue tasks with priorities, but Judoscale will be unaware of those priorities. Autoscaling is partitioned only on the queue names.

I think in an earlier conversation you said that if a task is enqueued without a queue name, it uses a default "celery" queue. Is that right?

If so, then if a user is _only_ using priorities for their tasks (no queue names), we'll still get queue time data, but it'll all be for a single "celery" queue. Sound right?

Reply:
Thanks for the explanation. That is unfortunate. 😕